python/ray/_private/worker.py

```diff
@@ -1648,7 +1648,22 @@ def color_for(data: Dict[str, str], line: str) -> str:
             else:
                 return colorama.Style.BRIGHT + colorama.Fore.CYAN
         else:
-            return colorama.Fore.CYAN
+            colors = [
+                colorama.Fore.BLUE,
+                colorama.Fore.CYAN,
```

Comment:
Can you remove CYAN since it can be used for the autoscaler log?

Reply:
This is done.